fcrepo-http-api/src/main/java/org/fcrepo/http/api/responses/StreamingBaseHtmlProvider.java

```diff
@@ -192,8 +192,14 @@ private Template getTemplate(final Model rdf, final Node subject,
 
         final String tplName = annotations.stream().filter(x -> x instanceof HtmlTemplate)
             .map(x -> ((HtmlTemplate) x).value()).filter(templatesMap::containsKey).findFirst()
-            .orElseGet(() -> multiValueURI(rdf.getResource(subject.getURI()), type).stream()
-                    .filter(templatesMap::containsKey).findFirst().orElse("default"));
+            .orElseGet(() -> {
+                final List<String> types = multiValueURI(rdf.getResource(subject.getURI()), type);
+                if (types.contains(REPOSITORY_NAMESPACE + "RepositoryRoot")) {
+                    return REPOSITORY_NAMESPACE + "RepositoryRoot";
+                } else {
```

Comment:
Yes, the `else` isn't needed here. If you are using Eclipse, there is definitely a setting to notify you about this kind of thing. I imagine other IDEs also can do that.